src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  margin-top: 20px;
+  font-family: "Avenir", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.input {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```

Comment:
```suggestion
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
  border-radius: 4px;
  outline: none
}
```
It will fix this
![2023-04-21_15h40_05](https://user-images.githubusercontent.com/61904995/233638421-a221d15e-a350-4040-9312-f89161eac59a.png)